src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
that stuff is a bit of a nightmare, if you remove the `any` it'll get typed for you from type inference ( you can test by removing it, then holding `alt` and clicking on fetchCollectorProfile.